### ISSUE TITLE
[MAS4.1.2]Label is not properly associated with controls defined under 'What color do you want'.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1348]
**Actual Result:**
When focus lands on 'What color do you want' combo box, radio buttons and check boxes then narrator is not reading label 'What color do you want' associated with the controls.
Labels are not associated with controls defined under 'What color do you want'.

Note - Same issue repro at - 
1. URL -> Help -> Show breakfast review using  Adaptive card -> Snooze for combo box - Label "Snooze for" is not associated with combo box.

**Expected Result:**
Labels should be associated with controls defined under 'What color do you want'.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'Help' in the input control at the bottom of the screen and hit enter (or click on send button).
4.  Content will upload, Navigate to any "Show an Adaptive cards with all types of inputs " button by using Tab key and press Enter on it.
5.  New data will upload.
6. Open Narrator (Ctrl + Win. key + Enter)
7. Navigate to 'combo box', 'radio buttons' and 'check boxes' by using Tab key.
8. Observe that Labels are not associated with controls defined under 'What color do you want'.

**User Impact:**
If Labels are not associated with controls then the users will get confuse about the purpose of that control.

**MAS Reference:**
MAS4.1.2 - Name,Role,Value

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** Narrator

**Attachment:**
[MAS4.1.2_Labels not associated with controls.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2579332/MAS4.1.2_Labels.not.associated.with.controls.zip)

**Comment by compulim** [comment]
This bug should belongs to Adaptive Cards project.

**Comment by corinagum** [comment]
Adaptive Cards issue can be found [here](https://github.com/Microsoft/AdaptiveCards/issues/2238).

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG4.1.2
